<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>761</width>
      <height>571</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QComboBox" name="tableChoose">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Выберите таблицу</string>
    </property>
    <item>
     <property name="text">
      <string>Таблица сотрудников</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Информация о сотрудниках</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Выплаты сотрудникам</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Отделы предприятия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Специальности</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="buttonUpdate">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>520</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить запись</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonInsert">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>480</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить запись</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDelete">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>560</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить запись</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1008</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionconnect_to_database">
   <property name="text">
    <string>connect to database</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
